src/localize/i18n.ts

```diff
@@ -0,0 +1,34 @@
+import i18n from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import Backend from 'i18next-xhr-backend';
+
+const detectionOptions = {
+  order: ['querystring', 'cookie', 'localStorage', 'sessionStorage', 'navigator', 'htmlTag'],
+  lookupQuerystring: 'lang',
+  lookupCookie: 'i18next',
+  lookupLocalStorage: 'i18nextLng',
+  lookupSessionStorage: 'i18nextLng',
+
+  // cache user language
+  caches: ['localStorage'],
+  excludeCacheFor: ['cimode'],
+};
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .init({
+    ns: ['app'],
+    defaultNS: 'app',
+    debug: true,
+    detection: detectionOptions,
+    fallbackLng: false,
+    interpolation: {
+      escapeValue: false,
+    },
+    backend: {
+      loadPath: '/locales/{{lng}}/{{ns}}.json'
```

Comment:
This kind of path will not work on gh-pages, and with the `/theme/` URLs, then add a todo comment here and in JIRA :-)